CONTRIBUTORS.md

```diff
@@ -15,3 +15,4 @@
 - [yorodm](https://github.com/yorodm)
 - [bollwyvl](https://github.com/bollwyvl)
 - [Dillan Mills](https://github.com/DillanCMills)
+- [Tom BH](https://github.com/tombh)
```

Comment:
It seems it was not always followed earlier either, but this list is supposed to be sorted in alphabetical order ( https://github.com/openlawlibrary/pygls/blob/f53d6615e4561184b393d4b2c6b5629f31f33dbb/CONTRIBUTORS.md#contributors-alphabetical )